### PR TITLE
Add support for DNSSEC

### LINKED ISSUE #786: Feature: DNSSec for domains
Add an option to secure domains with DNSSec

### PR DESCRIPTION
**Pull request by jaapmarcus**
Please note this PR is not stable and should not be used in production

Due the price raises for .ch domain without DNSSEC support it made sense to add support for it.

Please note Syncing "Hestia" API will not get supported due to issues with it. 

To enable it change manually edit DNSSEC in /usr/local/hestia/data/user/{user}/dns.conf and set DNSSEC='no' to 'yes' or add "DNSSEC='yes'

Then rebuild it.

See update information on how to enable it in combination with zone transfer below...

### CODE REVIEW

File: func/rebuild.sh [hunk 1]
```diff
@@ -517,7 +518,12 @@ rebuild_dns_domain_conf() {
         fi
         suspended_dns=$((suspended_dns + 1))
     else
-        if [ -z "$(grep /$domain.db $dns_conf)" ]; then
+        sed -i "/dns\/weetjes.net.db/d" $dns_conf
```
Comment (ScIT-Raphael) [inline func/rebuild.sh]: hardcoded domain name left :)

**Review by ScIT-Raphael — CHANGES_REQUESTED** [review]

**Comment by jaapmarcus** [comment]
Todo list

1. Restore Backup
2. Syncing 
3. Idn support / check
4. general testing 
5. Web UI

**Comment by jaapmarcus** [comment]
UI It self might need some ❤️ 

Screenshot 2022-10-02 at 21 48 32

**Comment by jaapmarcus** [comment]
**Do not update to this branch if your server contains critical domains**

Install branch:
Run `v-update-sys-hestia-git jaapmarcus feature/dnssec-support`

If you are planning to use it under a "single" setup without any "DNS cluster" just enable DNSSEC and run `v-list-dnssec-public-key user domain.com plain DS ` or `v-list-dnssec-public-key user domain.com plain` to get the required DS or DNSKEY 

For setting up a DNS Cluster:
On the "master" 

Run `v-update-sys-hestia-git jaapmarcus feature/dnssec-support`

nano /usr/local/hestia/conf/hestia.conf

And edit: DNS_CLUSTER_SYSTEM='hestia' to DNS_CLUSTER_SYSTEM='zone'

Open /etc/bind/named.options  and change the following:

`       allow-transfer {"none";};`
Remove "none" and add your slaves ip separated by a ;
Add 
`also-notify { slaveip; };`

On your slave:
Run `v-update-sys-hestia-git jaapmarcus feature/dnssec-support`

Open /etc/bind/named.options  and change the following:

`       allow-recursion { 127.0.0.1; ::1;};;`
Add masterip; 

Add 
`allow-notify{ masterip; };`

Now setup the slave as you would normally do.

**Review by ScIT-Raphael — APPROVED** [review]
Testing went properly, couldnt find any bug.

**Pull request merged**